### DISCUSSION TITLE
Welcome to Thrive Discussions!

### DISCUSSION THREAD

**Discussion by hhyyrylainen** [Revolutionary-Games/Thrive#1897] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to get help and discuss code related matters that don't belong to an issue:
  * Ask questions you’re wondering about.
  * Talk about things that should maybe be opened as issues
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
Can non-developers use this too as a alternative to the community forum for question pertaining to development?

**Reply by hhyyrylainen** [reply]
No, unless it is a very specific coding related question, refer to:
> to get help and discuss code related matters that don't belong to an issue

**Comment by DartPower** [comment]


**Reply by hhyyrylainen** [reply]
This isn't the place to ask or discuss that.

**Comment by span4ev** [comment]
when I thought that I could chill, while listening to an audiobook or watching a TV series in the background, and ended up feeling like a jerk after 2 minutes of the game, because you don't know a damn thing about biology.... Too bad there isn't a mode so you don't have to read and think about what you read, but just float and evolve carefree. Deleted.

**Comment by CI09** [comment]
Hello. I made feature where save manager has buttons for filtering between various save types (autosaves, quicksaaves, etc), but I don't know whether I should send a draft PR or not.

**Reply by ender-titan1** [reply]
If it’s ready to be reviewed (complete), make a normal PR, if not make a draft

**Comment by hhyyrylainen** [comment]
I'll close this discussion to avoid people asking unrelated things here rather than posting a new discussion about any code topics they would like to discuss.

Edit: actually I'll just lock this but keep it open so it stays visible, I hope to new people.
